network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -11,13 +11,13 @@
   <input type="hidden" class="parent-title" value="{{current_category.parent.localized.name}}">
   <div class="tw-container">
     <div class="tw-row">
-      <div class="tw-px-4 tw-w-full">
-        <div id="sticky-bar" class="creepiness-slider bg-white text-center tw-flex tw-justify-center">
+      <div class="tw-w-full">
```

Comment:
Can you add some context why we need these to be two elements? Isn't a row already full width?

Reply:
Rows in bootstrap are only responsible for adding flex and the negative margin. You always need a column class or to establish width else it will be width:auto, which is not what we want in this situation.

Reply:
Thanks @fessehaye this is helpful. 

Just wondering, do we still need to follow that Bootstrap convention of container-row-col? I can't really find any equivalent of the Bootstrap `col` classes for use with Tailwind.

Reply:
https://tailwindcss.com/docs/width Since we have `x/12` widths on tailwind. I do not really think adding the bootstrap columns back add any real value.

Reply:
True, but aren't those columns what creates the gutters between columns? I guess the question is, are the rows useful without the cols?

Reply:
It looks like Tailwind creates the space between row elements with `space-x-...`. 

![Uploading Screen Shot 2022-10-27 at 14.26.33.png…]()

I wonder if that negative margin + padding hack for the columns as in Bootstrap is still needed.

Reply:
So the trick is col-4 in bootstrap is the same as px-4 w-4/12 in tailwind. If you just add px-4 to column like elements it will cover the padding/negative margin. I documented this on the tailwind epic!